transformer_engine/common/common.h

```
@@ -307,10 +307,6 @@ struct TypeExtrema<fp8e4m3> {
   static constexpr float max = 448.0f;
 #elif defined(__HIP_DEVICE_COMPILE__)
   static constexpr float maxNorm = te_fp8_fnuz() ? 240.0f : 448.0f;
-#elif defined(TE_DYNAMIC_HIP_FP8_TYPE)
- // dummy declaration for correct translation;
- // it is not defined anywhere and it's usage should be eliminated before linking
-  static float maxNorm;
 #else
   static constexpr float maxNorm = 240.0f;
```

It should be dummy value here, so if at some moment maxNorm is accessed in host path, linking will fail

---

I have changed to be a dummy variable here, but I am curious why linking would fail if used hostside. Wouldn't maxNorm be zero-initialized if accessed since it is static?